### PR TITLE
add sklearn traintestsplitter

### PR DESCRIPTION
**Pull request by lvaleriu**

### CODE REVIEW

**Comment by sgugger** [comment]
I am now super familiar with this function from scikit-learn. Could you explain what it adds on top of our RandomSplitter?

**Comment by lvaleriu** [comment]
It adds the stratify argument that allows splitting uniformely according to the label distribution. Should I add this argument into the notebook too?

**Comment by sgugger** [comment]
Yes, it's best if the user can see all the arguments at their disposal. Don't know if @delegates is useful here or if scikit-learn hides things in kwargs.

**Comment by lvaleriu** [comment]
I’ll add the most common arguments then (like random_state, stratify, test_size)

**Comment by sgugger** [comment]
Can you please put the second line of documentation in a markdown cell of the notebook? We like the docstrings to be one line only, and all additional prose to be in the notebook itself.

**Comment by lvaleriu** [comment]
I had a bad feeling about it too and was hesitating to ask how you do when the docstrings are long :)

**Comment by sgugger** [comment]
Thanks a lot, this is looking good!
